ntbs-service/Jobs/ReportingDataProcessingJob.cs

```diff
@@ -74,24 +56,17 @@ protected virtual async Task<IEnumerable<dynamic>> ExecutePopulateForestExtractS
             return result;
         }
 
-        protected override bool DidExecuteSuccessfully(System.Collections.Generic.IEnumerable<dynamic> resultToTest)
+        protected override bool DidExecuteSuccessfully(IEnumerable<dynamic> resultToTest)
         {
-            var success = false;
-
             var serialisedResult = JsonConvert.SerializeObject(resultToTest);
             Log.Information(serialisedResult);
             _context.WriteLine($"Result: {serialisedResult}");
 
-            if (resultToTest.Count() == 0)
+            if (!resultToTest.Any())
             {
-                success = true;
+                return true;
             }
-            else
-            {
-                throw new ApplicationException("Stored procedure did not execute successfully as result has messages, check the logs.");
-            }
-
-            return success;
+            throw new ApplicationException("Stored procedure did not execute successfully as result has messages, check the logs.");
```

Comment:
This is just a personal matter of style, but I think that:
```
if (errorCondition)
{
    throw;
}
return true;
```
is easier to parse than:
```
if (noErrorCondition)
{
    return true;
}
throw;
```
Feel free to either update or not depending on whether you agree.